plate_mapper/plate_mapper.py

```diff
@@ -54,8 +55,9 @@ def plate_mapper(input_f, barseq_f, output_f, names_f=None, special_f=None):
         Sample name list file
     special_f : file object (optional)
         Special sample definition file
+    empty : bool (optional)
```

Comment:
how do you define empty? I got a mapping file from Greg containing lines that are composed of a lot of \t but no real characters.